console/command_in_controller.rst

```diff
@@ -50,11 +50,11 @@ Run this command from inside your controller via::
             ));
 
             // You can use NullOutput() if you don't need the output
-            $output = new BufferedOutput();
-            $application->run($input, $output);
+            $bufferedOutput = new BufferedOutput();
```

Comment:
I disagree with this renaming. The fact that the output is buffered is just an implementation detail that can change at any time. The important thing is that this is a console output, so `$output` looks OK to me.

Reply:
Agree, will be reverted.